ee/tabby-webserver/src/service/auth.rs

```diff
@@ -153,24 +156,8 @@ impl AuthenticationService for DbConn {
         input.validate()?;
 
         let is_admin_initialized = self.is_admin_initialized().await?;
-        let invitation = if is_admin_initialized {
```

Comment:
i don't think you need to modify `register` anymore? since the logic is user have to `request_invitation` in frontend to verify the email address.